tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -435,7 +435,10 @@ binary path.
 #### Install other dependencies
 
 ```bash
-$ sudo apt-get install python-numpy swig python-dev
+For Python 2.7:
```

Comment:
Can you add a # in front of the lines that are not commands on the command line?